crawl-ref/source/tilepick.cc

```diff
@@ -2481,6 +2481,7 @@ static tileidx_t _tileidx_missile_base(const item_def &item)
         }
 
     case MI_SLUG:
+        return TILE_BOLT_IRON_SHOT;
```

Comment:
I don't think this matters (these aren't real items), but I'd be dubious about using iron shot as an item tile if it did matter.

(This whole section should probably be cleaned up.)